cmd/report.go

```diff
@@ -286,17 +292,34 @@ func writeDetailedCSV(records []*reporting.CookbookRecord) {
 }
 
 func writeNodeReport(records []reporting.NodeReportItem) {
+
+	termWidth, _, err := terminal.GetSize(int(os.Stdout.Fd()))
```

Comment:
does his terminal go package works cross-platform?

Reply:
As we're using it, yes for Windows: https://github.com/golang/go/issues/20388.

We'll need to verify on mac, but since the mac support ioctl, I'd be surprised if it didn't.